Cargo.toml

```diff
@@ -19,4 +19,5 @@ wasmi = "0.5.0"
 lazy_static = "1.4.0"
 
 [features]
+debug = []
```

Comment:
Making this feature optional will mean you have to recompile whatever EE you're using, to toggle it on and off?

Might make more sense to toggle it at runtime by not setting a logging function?

Reply:
> Making this feature optional will mean you have to recompile whatever EE you're using, to toggle it on and off?

I think you'll have to recompile anyways, since you wouldn't want to deploy an EE with an invalid import (I think the wasm validation actually fails in that case), *but* I think you're right that toggling this at run time makes more sense than a compilation flag. I'm changing it to this and it feels like I should just use a real logger like `slog`.

Reply:
That's very true. I don't think runtime toggling is hard to implement though.